spec/requests/service_plans_spec.rb

```diff
@@ -98,4 +98,32 @@
       expect(json["schema"]).to eq service_plan.base["schema"]
     end
   end
+
+  describe "#modified" do
+    context "when there is a modified schema" do
+      before do
+        get "#{api}/service_plans/#{service_plan.id}/modified", :headers => default_headers
+      end
+
+      it "returns a 200" do
+        expect(response).to have_http_status :ok
+      end
+
+      it "returns the base schema from the service_plan" do
```

Comment:
Initially this is true, I wonder if the wording of this should change?  Modified won't always == base.

Reply:
The main reason I have it this way is just because in the factory I'm setting modified = base, so it will always return the same. This is just making sure it's returning the right object. I can change it in the factory so they aren't the same.